### PR TITLE
Update submodules + minor changes

### PR DESCRIPTION
**Pull request by scribam**
After merge, run these commands to update your submodules
```
git submodule sync
git submodule update --init --recursive
```

### CODE REVIEW

**Comment by Zangetsu38** [comment]
You can squash all "external: " in one commit for submodule updated ? 
Like "external: update some submodule"

**Comment by scribam** [comment]
Probably not a good idea if I have to bisect in case of regressions 🤔

**Comment by Zangetsu38** [comment]
I see but 17 commit just for uldare submodule hmhm i no very like lol you can simple set this external: in one

File: vita3k/util/include/util/log.h [hunk 1]
```diff
@@ -69,5 +71,3 @@ std::string log_hex(T val) {
     using unsigned_type = typename std::make_unsigned<T>::type;
     return fmt::format("0x{:0X}", static_cast<unsigned_type>(val));
 }
-
```
Comment (Zangetsu38) [inline vita3k/util/include/util/log.h:72]: keep new line in end file

**Review by Zangetsu38 — CHANGES_REQUESTED** [review]

File: vita3k/util/include/util/log.h (hunk 1 above)
Comment (scribam) [reply vita3k/util/include/util/log.h:72]: It has one now. It was previously missing.
A missing line at the end of a file has the sign sign and I removed it.

**Comment by scribam** [comment]
You can "Squash and merge". This way, the branch keeps all the commits and master will only have one.

File: vita3k/util/include/util/log.h [hunk 2]
```diff
@@ -69,5 +71,3 @@ std::string log_hex(T val) {
...
-void dump_hex(const std::vector<uint8_t> &bytes, std::ostream &stream);
```
Comment (Zangetsu38) [inline vita3k/util/include/util/log.h:73]: I speek here end file finish by } and no have new line

**Review by Zangetsu38 — CHANGES_REQUESTED** [review]

**Comment by Zangetsu38** [comment]
> You can "Squash and merge". This way, the branch keeps all the commits and master will only have one.

Oki

File: vita3k/util/include/util/log.h (hunk 2 above)
Comment (scribam) [reply vita3k/util/include/util/log.h:73]: Yes I know and it has one. A missing line at the end of a file has the sign image or do you want one more empty line?

Comment (Zangetsu38) [reply vita3k/util/include/util/log.h:73]: All end file request one new line after code

Comment (scribam) [reply vita3k/util/include/util/log.h:73]: Yes and it has one. I can't add an other line, clang-format doesn't want it.

Comment (Zangetsu38) [reply vita3k/util/include/util/log.h:73]: Oh strange

**Review by Zangetsu38 — APPROVED** [review]

**Pull request merged**